doc/source/whatsnew/v2.1.0.rst

```diff
@@ -71,9 +71,10 @@ to ``na_action=None``, like for all the other array types.
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
-- :meth:`Categorical.map` and :meth:`CategoricalIndex.map` now have a ``na_action`` parameter.
-  :meth:`Categorical.map` implicitly had a default value of ``"ignore"`` for ``na_action``. This has formally been deprecated and will be changed to ``None`` in the future.
+- :meth:`Categorical.map` implicitly had a default value of ``"ignore"`` for ``na_action``. This has formally been deprecated and will be changed to ``None`` in the future.
```

Comment:
Why are a lot of unrelated entries modified in this diff? I think only one entry should have been added

Reply:
@mroeschke Mistakes caused by handling merge conflicts. They should have been fixed from previous comments left by reviewers. Are we still seeing issues?

Reply:
Ideally for this pull request we should just see one new entry but other entries in this diff appear changed

Reply:
@mroeschke I'll pull the latest version of `whatsnew/v2.1.0.rst` from master, add my change for this PR and push. That should guarantee that all incorrect modifications to the file have been fixed

Reply:
@mroeschke Updated `whatsnew/v2.1.0.rst` should be good